source/game/system/CourseMap.cc

```diff
@@ -233,6 +233,372 @@ CourseMap::~CourseMap() {
     delete m_stageInfo;
 }
 
+/// @addr{0x80511E00}
+void CourseMap::clearSectorChecked() {
+    for (size_t i = 0; i < m_checkPoint->size(); ++i) {
+        getCheckPoint(i)->clearSearched();
+    }
+}
+
+/// @addr{0x80511500}
+s16 CourseMap::findSector(const EGG::Vector3f &pos, u16 checkpointIdx, f32 &distanceRatio) {
+    clearSectorChecked();
+
+    MapdataCheckPoint *checkpoint = getCheckPoint(checkpointIdx);
+    s16 id = -1;
+
+    MapdataCheckPoint::SectorOccupancy completion =
```

Comment:
```suggestion
    MapdataCheckPoint::SectorOccupancy occupancy =
```
The original name is a holdover from decompilation.